pydbtools/__init__.py

```diff
@@ -31,4 +31,4 @@
 )
 from .utils import s3_path_join  # noqa: F401
 
-__version__ = "5.6.2"
+__version__ = "5.6.3"
```

Comment:
So instead of hard coding this twice, you can use 

```python
from importlib.metadata import version

__version__ = version(pydbtools)  # this is the package name as defined in the pyproject.toml
```